tests/SessionTest.php

```diff
@@ -83,34 +101,10 @@ public function testRestart()
         $this->session->restart();
     }
 
-    public function testVisitWithoutRunningSession()
```

Comment:
@stof , @stof , any updates? This seem to be last change to be made before merging.

P.S.
Change actually needs to be made on #732 instead and this PR rebased on top of it.